community/planning/queuer_strat_one_round_serial.md

```diff
@@ -0,0 +1,103 @@
+# One-round serial queuer strategy
```

Comment:
Need to add the copyright to this file

Reply:
Also need to add a link to this doc somewhere, it's inaccessible atm

Reply:
Fixed, thanks!